proxy/core/src/main/scala/io/cloudstate/proxy/streams/MergeSequence.scala

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2019 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudstate.proxy.streams
+
+import akka.NotUsed
+import akka.stream.{Attributes, Graph, Inlet, Outlet, UniformFanInShape}
+import akka.stream.stage.{GraphStage, GraphStageLogic, InHandler, OutHandler}
+
+import scala.collection.mutable
+
+object MergeSequence {
```

Comment:
Didn't this already get added to akka?

Reply:
Yes it did, but my initial aim was just to get the branch rebased and compiling, which is what this is.